pycloudlib/lxd/instance.py

```diff
@@ -13,6 +13,12 @@ class LXDInstance(BaseInstance):
     _type = 'lxd'
     _is_vm = None
 
+    _missing_agent_msg = (
+        "Many Xenial images do not support `%s` due to missing lxd-agent:"
+        " you may see unavoidable failures.\n"
+        "See https://github.com/canonical/pycloudlib/issues/132 for details."
+    )
+
```

Comment:
This is fine here, but it could probably just be a module-level constant: we don't need to be able to modify it within the inheritance hierarchy here.

Reply:
Done, let me know if you had anything different in mind.

(I'm doing separate commits for these fixups, we'll squash before merging.)